torch/_ops.py

```diff
@@ -46,6 +46,12 @@ def __call__(self, *args, **kwargs):
     def __getattr__(self, key):
         return getattr(self._op, key)
 
+    def __eq__(self, other):
+        if self._op == other._op:
```

Comment:
nit: is self._op "the pointer to a specific operator overload"? If so, it's not clear from the variable name -- self._op_overload may be clearer or maybe a comment next to `self._op = op # pointer to specific operator overload` would help.